lib/addons/vpc-cni/eniConfig.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: crd.k8s.amazonaws.com/v1alpha1
```

Comment:
Must be renamed to yptl - we had that issue before and quickstart yaml check is failing.

Reply:
Done.